app/models/ability.rb

```diff
@@ -66,8 +66,7 @@ def initialize(user)
     end
 
     can :export, Group do |group|
-      user.is_admin? or
```

Comment:
i guess it was because it's an undocumented thing.

Happy to put it back in if you like. Sorry for the inconvenience.